### ISSUE TITLE
Unable to use decoded NEC remote to output through UNO digital pins.

### ISSUE THREAD

**Issue by jimercer** [z3t0/Arduino-IRremote#517]
**Board:** ARDUINO UNO
**Library Version:** 2.1.0
**Protocol:** NEC

At this time I am trying to use your library with my Arduino UNO to use an IR remote control to turn led's on and off. 
I used the library previously to determine all of the codes from the remote control that I will be using. Your library determined that all of the codes were NEC codes. 
The decode program provided the format of FF1234. I tried to use this format in my sketch code but the complier hates it but allows the format prefixed with the 0x, ie: 0xFF1234.

That being said, I have no problems receiving data from the remote. However, I am unable to do anything with the data received. 
in the void loop after the,
if (irrecv.decode(&results))
statement, I have tried using the results by first defining the code, ie:   #define CODE_FF 0xFFC23D //code from IR remote button “Fast Forward”, and then using the CODE_FF in the next statement,
if(results.value == CODE_FF){
digitalWrite(pin, HIGH)....

I have also tried using the specific code ie: 0xFF1234 or FF1234 in place of the CODE_FF. I have tried everything I can think of,  but nothing I try seems to connect the "results.value" statement to the actual input of the remote.
I have created 5 different sketches using various methods to simply use the remote to turn on an led by using a digital pin, (of course first assigning the pin as a, 
int RECV_PIN = 2;
int LED_PIN1 = 3;
pinMode(RECV_PIN, INPUT);
pinMode(LED_PIN1, OUTPUT);
,but nothing seems to work.  

I have looked through all of the library for the public variable definitions and don't see anything about this use. 
In short, is calling "results.value" the correct call for the input data from the decoder? Am I missing something?

P.S. I can provide my sketches, but didn't think it necessary as they compile fine, just no communicaiton occurs which tells me it's a problem with the language used in the library and not the format, math, or other sketch code issue. 

P.P.S. HELP ME PLEASE!!!
